DatadogInternal/Sources/NetworkInstrumentation/URLSession/DatadogURLSessionDelegate.swift

```diff
@@ -12,12 +12,6 @@ public typealias DDURLSessionDelegate = DatadogURLSessionDelegate
 /// The implementation must ensure that required methods are called on the `ddURLSessionDelegate`.
 @objc
 public protocol __URLSessionDelegateProviding: URLSessionDelegate {
```

Comment:
keeping this type for not breaking customers but essentially it is of no use. The customer delegate is swizzled and we do the right thing. There is no need to forward delegate calls.

Reply:
Maybe we should mark it deprecated and remove on next major release?

Reply:
separate PR as it is tricky, because we treat warning as errors in tests.